cinema/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)
+    description = serializers.CharField(required=False, max_length=500)

```

Comment:
i think required=False is set by default

Reply:
No, on the contrary)